src/__utils/randomId.ts

```diff
@@ -0,0 +1,3 @@
+export function randomId(seed: string): string {
+    return seed + String((Math.random() * 1000) + (new Date()).getTime());
```

Comment:
It's safe to use string interpolation instead of String constructor as they work the same way.
example ``${null} ${undefined}`` will produce a string `"null undefined"` just like `String(null) + " " + String(undefined)` would.

```suggestion
    return `${seed}${Math.random() * 1000}${new Date()).getTime()}`;
```

Reply:
You just removed that mathematical operation that sums the `Math.random() * 1000` and `new Date()).getTime()` before converting them into `string`. Your solution will increase its foot print. For example:
```javascript
console.log(348723 + 2937849); // Result: 3286572
console.log(`${348723}${2937849}`); // Result: 3487232937849
```
I don't see a need for that